cloud/ecs-integration.md

```diff
@@ -201,14 +201,20 @@ docker secret create dockerhubAccessToken --username <dockerhubuser>  --password
 arn:aws:secretsmanager:eu-west-3:12345:secret:DockerHubAccessToken
 ```
 
-Once created, you can use this ARN in you Compose file using using `x-aws-pull_credentials` custom extension with the Docker image URI for your service.
+Once created, you can use this ARN in you Compose file using using `x-aws-pull_credentials` custom extension with the Docker image URI for your service. For AWS ECR Registry private repository, use `x-aws-pull_policy` to authenticate.

```

Comment:
We configure ECS Task with ECRReadAccess permission so this should **not** be required. Please open an issue on docker/compose-cli if you have to

Reply:
reference to `x-aws-pull_policy` is a mistake, we don't define such an extension
https://github.com/docker/compose-cli/blob/main/ecs/x.go#L20-L33